src/main.rs

```diff
@@ -128,15 +102,34 @@ async fn launch_web() {
         };
     }
 
-    let auth = Auth::new(database::get_db(), config);
+    let db = DatabaseInfo::Auto.connect().await.unwrap();
+    db.migrate_database().await.unwrap();
+
+    let mongo_db = mongodb::Client::with_uri_str("mongodb://localhost")
```

Comment:
Is this meant to be a static string instead of the `MONGODB` environment variable?